appveyor-clean-cache.txt

```diff
@@ -0,0 +1 @@
+1
```

Comment:
It wasn't originally clear to me what the purpose of this file was; I believe the idea is that we can trigger an Appveyor cache clean by modifying the contents of this file. I'll add a comment to that effect.